src/components/widgets/ChartWidget.tsx

```
@@ -288,91 +716,183 @@ export function ChartWidget({ symbol }: { symbol?: string | undefined } = {}) {
+              Clear
+            </button>
+          )}
+        </div>
+      </div>
+
+      <div
+        ref={chartAreaRef}
+        className="relative flex-1"
+        onContextMenu={handleContextMenu}
+        data-testid="chart-surface"
+      >
```

Biome (2.4.7)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)